content/docs/command-reference/root.md

```diff
@@ -10,11 +10,15 @@ usage: dvc root [-h] [-q | -v]
 
 ## Description
 
-This command returns the path to the root directory of the <abbr>DVC
-project</abbr>, relative to the current working directory. It can be used to
-build a path to a dependency, script, or <abbr>data artifact</abbr>, for
-example. Useful when working in a subdirectory of the project, and needing to
-refer to a file in another directory.
+The `dvc root` command returns the path to the root directory of the <abbr>DVC
```

Comment:
Let's keep it short :)
```suggestion
Returns the path to the root directory of the <abbr>DVC
```

Your version is correct but we try to avoid repetition and especially for `dvc {cmd}` because those get linked automatically to the cmd ref. We don't need so many links back to the same web page.

Reply:
This change will require reformatting the paragraph.

Reply:
Removed extra links and shortened first sentence as suggested.

Reply:
Thanks. Could've left one instance 😅 but it's OK either way.